tfx/components/transform/component.py

```diff
@@ -74,8 +74,11 @@ def __init__(
       transform_graph: Optional[types.Channel] = None,
       transformed_examples: Optional[types.Channel] = None,
       input_data: Optional[types.Channel] = None,
+      cache_input_path: Optional[types.Channel] = None,
+      cache_output_path: Optional[types.Channel] = None,
       instance_name: Optional[Text] = None,
       materialize: bool = True,
+      materialize_cache: bool = True,
```

Comment:
How about `store_cache` instead of materialize? I think that store makes more sense in the context of cache

Reply:
updated, thanks!